airflow/providers/google/cloud/operators/dataproc_metastore.py

```diff
@@ -29,13 +29,139 @@
+                region=backups_list_conf["region"],
+                service_id=backups_list_conf["service_id"],
+                project_id=backups_list_conf["project_id"],
+            )
+            if backups_list_conf
+            else ""
+        )
+
+
+class DataprocMetastoreExportLink(BaseOperatorLink):
+    """Helper class for constructing Dataproc Metastore Export Metadata link"""
+
+    name = "Dataproc Metastore Export Metadata"
+
+    def get_link(self, operator, dttm):
+        ti = TaskInstance(task=operator, execution_date=dttm)
+        export_conf = ti.xcom_pull(task_ids=operator.task_id, key="export_conf")
+        return (
+            METASTORE_EXPORT_LINK.format(
+                region=export_conf["region"],
+                service_id=export_conf["service_id"],
+                project_id=export_conf["project_id"],
+            )
+            if export_conf
+            else ""
+        )
+
+
+class DataprocMetastoreImportLink(BaseOperatorLink):
+    """Helper class for constructing Dataproc Metastore Import Metadata link"""
+
+    name = "Dataproc Metastore Import Metadata"
+
+    def get_link(self, operator, dttm):
+        ti = TaskInstance(task=operator, execution_date=dttm)
+        import_conf = ti.xcom_pull(task_ids=operator.task_id, key="import_conf")
+        return (
+            METASTORE_IMPORT_LINK.format(
+                region=import_conf["region"],
+                service_id=import_conf["service_id"],
+                import_id=import_conf["import_id"],
+                project_id=import_conf["project_id"],
+            )
+            if import_conf
+            else ""
+        )
+
+
+class DataprocMetastoreServiceLink(BaseOperatorLink):
+    """Helper class for constructing Dataproc Metastore Service link"""
+
+    name = "Dataproc Metastore Service"
+
+    def get_link(self, operator, dttm):
+        ti = TaskInstance(task=operator, execution_date=dttm)
+        service_conf = ti.xcom_pull(task_ids=operator.task_id, key="service_conf")
+        return (
+            METASTORE_SERVICE_LINK.format(
+                service_id=service_conf["service_id"],
+                region=service_conf["region"],
+                project_id=service_conf["project_id"],
+            )
+            if service_conf
+            else ""
+        )
+
+
+class DataprocMetastoreStorageLink(BaseOperatorLink):
+    """Helper class for constructing GCS Storage link for Dataproc Metastore Export Metadata"""
+
+    name = "GCS Storage for Dataproc Metastore Export"
+
+    def get_link(self, operator, dttm):
+        ti = TaskInstance(task=operator, execution_date=dttm)
+        storage_conf = ti.xcom_pull(task_ids=operator.task_id, key="storage_conf")
+        return (
+            GCS_STORAGE_LINK.format(
+                destination_gcs_uri=storage_conf["destination_gcs_uri"],
+                project_id=storage_conf["project_id"],
+            )
+            if storage_conf
+            else ""
+        )
```

Comment:
Is it possible to create one generic Store link class and use it in other operators?